source/georef/street_network.h

```diff
@@ -32,6 +32,8 @@ www.navitia.io
 #include "georef.h"
 #include "dijkstra_path_finder.h"
 #include "astar_path_finder.h"
+#include "routing/raptor_utils.h"
+#include "type/type.h"  //TODO: Remove
```

Comment:
minor: Poke in case you forgot this (but I guess it's for a later PR).

Reply:
it's for later :cry: 
A lot of things broke once I started to remove type.h...